packages/block-library/src/gallery/gallery-image.js

```diff
@@ -128,6 +128,25 @@ class GalleryImage extends Component {
 		/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/onclick-has-role, jsx-a11y/click-events-have-key-events */
 		return (
 			<figure className={ className } tabIndex="-1" onKeyDown={ this.onKeyDown } ref={ this.bindContainer }>
+				{ process.env.GUTENBERG_PHASE === 2 ?
```

Comment:
I think we should remove the check, as it's a good enhancement to land in Core regardless.

Reply:
Yeah, I'm perhaps being overly cautious here. 50172ee